model/component.js

```diff
@@ -208,7 +208,7 @@ class Component extends ParchKey {
         let valid = true;
 
         if (!this.feature) {
-            console.log('Component (WARNING): Field "feature" has not been set.');
+            console.log('Component (' + this.name + ') (WARNING): Field "feature" has not been set.');
```

Comment:
If its a warning just use [`console.warn()`](https://developer.mozilla.org/en-US/docs/Web/API/Console/warn)

Reply:
It's been changed.